### PR TITLE
Create Kodiak config.yaml

### PR DESCRIPTION
**Pull request by thedoc31**
* Create Kodiak auto-ticketing config for MWPW-143712 and MWPW-143708

Resolves:
- [MWPW-143712](https://jira.corp.adobe.com/browse/MWPW-143712)
- [MWPW-143708](https://jira.corp.adobe.com/browse/MWPW-143708)

**Test URLs:**
- Before: https://main--cc--adobecom.hlx.page/?martech=off
- After: https://thedoc31-patch-1--cc--adobecom.hlx.page/?martech=off

### CODE REVIEW

**Review by salonijain3 — APPROVED** [review]

File: .kodiak/config.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+notifications:
+  jira:
+    default:
+      project: MWPW # Mandatory
+      filters:
+        include:
+          risk_rating: R5
+      fields:
+        assignee:
+          name: seanchoi
```
Comment (honstar) [inline .kodiak/config.yaml:15]: Should all milo cc tickets be assigned to @seanchoi-dev automatically? Not sure this scales over time.

Comment (aishwaryamathuria) [reply .kodiak/config.yaml:15]: +1. Could we get members from CC team assigned to the tickets?
@salonijain3

Comment (thedoc31) [reply .kodiak/config.yaml:15]: @honstar Unfortuantely Kodiak does not currently allow assigning to a group, but Sean can delegate as needed. If there's a different individual who is better suited to do this, I'm happy to assign them.

@aishwaryamathuria Can add anyone you'd like, just need to know who else should be watchers on the ticket.

Comment (dstrong23) [reply .kodiak/config.yaml:15]: @thedoc31 Can you please add @salonijain3 and myself as watchers? One of us get make sure that these tickets get assigned.

Comment (salonijain3) [reply .kodiak/config.yaml:15]: Also, @thedoc31 you can assign all tickets to <suman h> ldap : <sumanh@adobe.com> for adobecom/cc and then we can reassign accordingly.

Comment (thedoc31) [reply .kodiak/config.yaml]: Alright, changes are committed @dstrong23 @salonijain3

**Pull request merged**